README.md

```diff
@@ -0,0 +1,32 @@
+# Flights Application
+
+Software application that is able to request all flight data from two APIs (<https://discovery-stub.comtravo.com/source1>) and (<https://discovery-stub.comtravo.com/source2>) and then it will send back to customer a response in JSON format of all flights but deleting duplicates in the process.
```

Comment:
"Software Application"
![](https://media.giphy.com/media/3bKZlDZjU0hR6/giphy.gif)

Reply:
got it haha

Reply:
Hahaha nah, I was just thinking that it looks so formal and fancy. All good